cli.py

```diff
@@ -22,34 +22,53 @@ def cli_load_parq (
     save_csv: str = typer.Option(None, "--save-csv", help="output as CSV"),
     save_rdf: str = typer.Option(None, "--save-rdf", help="output as RDF"),
     rdf_format: str = typer.Option("ttl", "--format", help="RDF format: ttl, rdf, jsonld, etc."),
+    encoding: str = typer.Option("utf-8", "--encoding", help="output encoding"),
+    dump: bool = typer.Option(False, "--dump", help="dump the data, only"),
+    sort: bool = typer.Option(False, "--sort", help="sort the output"),
     debug: bool = False,
     ) -> None:
     """
 Load a Parquet file into a graph partition, optionally converting and
 saving to different formats.
     """
-    parq_file: pq.ParquetFile = pq.ParquetFile(load_parq)
-
-    if debug:
-        ic(parq_file.metadata)
-        ic(parq_file.schema)
-        ic(type(parq_file.schema))
-
     part: Partition = Partition(
         part_id = 0,
     )
 
-    part.parse_rows(part.iter_load_parquet(parq_file))
+    parq_file: pq.ParquetFile = pq.ParquetFile(load_parq)
+
+    # in this case, only print what Parquet has parsed then quit
+    if dump:
+        part.dump_parquet(parq_file)
+        return
+
+    part.parse_rows(
+        part.iter_load_parquet(
+            parq_file,
+            debug = debug,
+        ),
+        debug = debug,
+    )
 
     if debug:
         ic(part)
 
     # next, handle the output options
     if save_csv is not None:
-        part.save_file_csv(cloudpathlib.AnyPath(save_csv))
+        part.save_file_csv(
+            cloudpathlib.AnyPath(save_csv),
+            encoding = encoding,
+            sort = sort,
+            debug = debug,
+        )
 
     if save_rdf is not None:
-        part.save_file_rdf(cloudpathlib.AnyPath(save_rdf), rdf_format)
+        part.save_file_rdf(
```

Comment:
Should we add `encoding` here also?  `kglab` uses `utf-8` also (https://github.com/DerwenAI/kglab/blob/48130af514826184cc42dc3e5f16aa489d5d846b/kglab/serde.py#L254-L263), but if the user chooses a different encoding it should be consistent between `csv` and `rdf`. This is very minor, though, and it might even be a good idea to remove encoding option. (not sure how important this is)

Reply:
Great, good catch!
Yes the encoding for RDF is important.
We've already run into some related issues with RDF imports, although those were more at the XSD type casting level (e.g., different float formats across between Spain and Germany).
RDF encoding may also become important for example in support for East Asian languages.